src/main/java/graphql/execution/ValuesResolver.java

```diff
@@ -82,25 +135,6 @@
     }
 
 
-    private Object getVariableValue(GraphQLSchema schema, VariableDefinition variableDefinition, Object inputValue) {
-        GraphQLType type = TypeFromAST.getTypeFromAST(schema, variableDefinition.getType());
-
-        //noinspection ConstantConditions
-        if (!isValid(type, inputValue)) {
-            throw new GraphQLException("Invalid value for type");
-        }
-
-        if (inputValue == null && variableDefinition.getDefaultValue() != null) {
-            return coerceValueAst(type, variableDefinition.getDefaultValue(), null);
-        }
-
-        return coerceValue(type, inputValue);
-    }
-
-    private boolean isValid(GraphQLType type, Object inputValue) {
-        return true;
-    }
-
```

Comment:
Got rid of the isValid since its does nothing and am looking to replace it with a call back system in a future PR